locales/sk/json.json

```diff
@@ -351,37 +351,37 @@
     "June": "Jún",
     "Kazakhstan": "Kazachstan",
     "Kenya": "Keňa",
-    "Key": "Tlačidlo",
+    "Key": "Kľúč",
     "Kiribati": "Kiribati",
     "Korea": "Južná Kórea",
     "Korea, Democratic People's Republic of": "Severná Kórea",
     "Korea, Republic of": "Kórea, Republika",
-    "Kosovo": "Kosovský",
+    "Kosovo": "Kosovo",
     "Kuwait": "Kuvajt",
     "Kyrgyzstan": "Kirgizsko",
     "Lao People's Democratic Republic": "Laos",
-    "Last active": "Posledná aktívna",
+    "Last active": "Naposledy aktívny",
     "Last used": "Naposledy použité",
     "Latvia": "Lotyšsko",
     "Leave": "Opustiť",
     "Leave Team": "Opustiť Tím",
     "Lebanon": "Libanon",
-    "length": "length",
-    "Lens": "Objektív",
+    "length": "dĺžka",
+    "Lens": "Pohľad",
     "Lesotho": "Lesotho",
     "Liberia": "Libéria",
     "Libyan Arab Jamahiriya": "Líbya",
     "Liechtenstein": "Lichtenštajnsko",
-    "Light": "Svetlo",
+    "Light": "Svetlý",
     "Lithuania": "Litva",
-    "Load :perPage More": "Načítať :perPage viac",
-    "Loading": "Načítava",
+    "Load :perPage More": "Načítať :perPage ďalších",
+    "Loading": "Načítavam",
     "Location": "Poloha",
     "Log in": "Prihlásiť",
     "Log In": "Prihlásiť sa",
-    "Log Out": "odhlásiť",
-    "Log out": "Odhlásiť sa",
-    "Log Out Other Browser Sessions": "Odhláste Sa Z Iných Relácií Prehliadača",
+    "Log Out": "Odhlásiť sa",
+    "Log out": "Odhlásiť",
```

Comment:
I think the translation meanings of the phrases "Log Out" and "Log out" should match, since they have the same meaning in Laravel.

They are written differently because they were written differently in the source code of different Laravel projects.

Reply:
Changed